.goreleaser.yml

```diff
@@ -5,6 +5,15 @@ release:
     name: kubeaudit
   draft: true
   name_template: "{{.ProjectName}}-v{{.Version}}"
+dockers:
+- dockerfile: goreleaser.Dockerfile
+  goos: linux
+  goarch: amd64
+  goarm: ''
+  image_templates:
+  - "ghcr.io/shopify/kubeaudit:latest"
+  - "ghcr.io/shopify/kubeaudit:{{ .Tag }}"
+  - "ghcr.io/shopify/kubeaudit:v{{ .Major }}.{{ .Minor }}"
```

Comment:
Noice: this pattern is cool!

Reply:
this is the pattern that we had before, but were pointing to docker hub registries instead